cluster-operator/src/main/java/io/strimzi/operator/cluster/model/VolumeUtils.java

```diff
@@ -405,4 +407,90 @@ private static String getVolumeNameHashStub(String name)   {
             throw new RuntimeException("Failed to get volume name SHA-1 hash", e);
         }
     }
+
+    /**
+     * Creates the Client Secret Volume
+     * @param tls    Kafka Client tls object
+     * @param volumeList    List where the volumes will be added
+     * @param isOpenShift   Indicates whether we run on OpenShift or not
+     */
+    protected static void createClientSecretVolume(ClientTls tls, List<Volume> volumeList, boolean isOpenShift) {
```

Comment:
It configured TLS encryption ... so what about `createTlsEncryptionSecretvolume` or `createCaSecretVolume` or something like that?